packages/api/src/shared/functions.ts

```diff
@@ -90,14 +90,15 @@ export const upload = async (req: Request, key: string): Promise<string> => {
 
     await logo.mv(path);
 
-    return path;
+    return filename;
   } catch (e) {
     console.log('ERROR:', e);
     throw new InternalServerError('File upload failed.');
   }
 };
 
-export const removeFile = async (filepath: string): Promise<void> => {
+export const removeFile = async (filename: string): Promise<void> => {
+  const filepath = `uploads/${filename}`;

```

Comment:
Do we want to pull out the definition of the upload directory `uploads/` into env configuration?  (or else just defined as a constant in this file)

Reply:
Fixed